Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+const CardContainer = (props) => {
+    return (
+        <a href="https://www.reactjs.org" target="_blank" className="cardContain">
```

Comment:
remember our zoom; we concluded this was a better way to link to a new page instead of adding an event listener through a querySelected  element